cli/src/test/resources/ide-projects/jasypt/repository/jasypt/jasypt/default/InstallTest.txt

```diff
@@ -0,0 +1 @@
+This is a test file.
```

Comment:
Here you would actually need to place a `jasypt-default.jar` file. It can have the same content and the faked/mocked `java` could read and echo the content of the JAR file provided after `-cp` so you can test that all is working.